.github/workflows/gradle.yml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
   schedule:
     - cron: '0 3 * * *'
+  workflow_dispatch:
```

Comment:
At what time do we need to trigger the workflow manually?

Reply:
Well, we never know in advance. For example, the daily build failed this morning (see https://github.com/acrolinx/sdk-demo-java/actions) and I wanted to quickly trigger another one to see if it just was a fluke (which I think it was), but I could not do it since we had not enabled this feature for this git repo. Therefore I think the best is to enable it for all our GitHub repos, so we have it ready in our toolbox in case we want to use it.